### PR TITLE
#7 Set up test Ruuter instance to pass requests to Turvis

### LINKED ISSUE #7: Set up test Ruuter instance to pass requests to Turvis
AS A Product Owner
I WANT test instances of Ruuter and Turvis to be set up and Ruuter passing all of its incoming requests to Turvis
SO THAT I would have a test playground to validate the current state of Turvis and its developments

### Acceptance Criteria

- [ ] Ruuter's requests passing functionality based on https://github.com/buerokratt/Ruuter/issues/9
- [ ] Use the latest version of [Ruuter](https://github.com/buerokratt/Ruuter)
- [ ] Use the latest version of [Turvis](https://github.com/buerokratt/Turvis)

#### Docker Compose

- [ ] Launch `turvis-test` on port `8500`
- [ ] Launch `turvis-ruuter-test` on port `8505`

### PR DESCRIPTION
**Pull request by hmargus**
Adds integration setup between ruuter and turvis. 

- Turvis and Ruuter setup under integration directory
- Ruuter passes the query to turvis

### CODE REVIEW

**Comment by hmargus** [comment]
this ticket needs following improvements:
- pass the env parameter so that the application knows if to return response body or not. In production, we don't want to see response codes
- enable building ruuter docker image based on specific tag/commit/branch

**Review by turnerrainer — APPROVED** [review]
Approved as initial commit

File: README.md [hunk 1]
```diff
@@ -16,12 +16,39 @@ The functionality is to map all the requests paths to generic endpoint in Turvis
...
   - Lint static checks
   - fastify node.js server
   - Dockerfile for building the image
   - docker-compose for running the service
-- two endpoints, `GET turvis:8060/ruuter-incoming`and `POST turvis:8060/ruuter-incoming`, for accepting calls from ruuter.
\ No newline at end of file
+- two endpoints, `GET turvis:8060/ruuter-incoming`and `POST turvis:8060/ruuter-incoming`, for accepting calls from ruuter.
+
+## feature/7
+- turvis and ruuter setup with one composer file. \
```
Comment (turnerrainer) [inline README.md:31]:
```suggestion
- Turvis and Ruuter setup with one composer file. \
```

File: README.md [hunk 2]
```diff
@@ -16,12 +16,39 @@ The functionality is to map all the requests paths to generic endpoint in Turvis
...
+   ```
+   cd integration
+   make start
+   ```
+   By default it will build ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```
Comment (turnerrainer) [inline README.md:36]:
```suggestion
   By default it will build ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```
```suggestion
   By default it will build Ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```

File: README.md [hunk 3]
```diff
@@ -16,12 +16,39 @@ The functionality is to map all the requests paths to generic endpoint in Turvis
...
+   - build a specific commit: \
+   `docker build --build-arg GIT_COMMIT=<commit_hash> -t buerokratt/ruuter-test -f ruuter/Dockerfile.ruuter  .`
+   - To build a specific tag: \
+   `docker build --build-arg GIT_TAG=<tag_name> -t buerokratt/ruuter-test -f ruuter/Dockerfile.ruuter  .`
+
+   The same configuration is exposed to docker-compose via `.env` file
+
+   for corner cases when there is a need to build from forked repo, one can be configured with `GIT_REPO_URL`
+
+- simple endpoint is configured for ruuter to forward requests to `turvis` \
```
Comment (turnerrainer) [inline README.md:53]:
```suggestion
- simple endpoint is configured for Ruuter to forward requests to `Turvis` \
```

**Review by turnerrainer — APPROVED** [review]
